tests-functional/clients/services/wakuext.py

```diff
@@ -0,0 +1,57 @@
+from clients.rpc import RpcClient
+from clients.services.service import Service
+from tenacity import retry, stop_after_delay, wait_fixed
+
+
+class WakuextService(Service):
+    def __init__(self, client: RpcClient):
+        super().__init__(client, "wakuext")
+
+    def send_contact_request(self, contact_id: str, message: str):
+        params = [{"id": contact_id, "message": message}]
+        response = self.rpc_request("sendContactRequest", params)
+        return response.json()
+
+    def accept_contact_request(self, chat_id: str, retry_timeout: int = 10):
```

Comment:
It's not actually a `chat_id`, it's the `message_id` of the contact request message. Which we usually call `request_id`:
https://github.com/status-im/status-go/blob/d291204473d2dcdb27062f017ffe9966463d4818/protocol/messenger_contacts.go#L100

```suggestion
    def accept_contact_request(self, request_id: str, retry_timeout: int = 10):
```

Reply:
Done